Assets/Scripts/ConvergeGame/MultiConvergeGame.cs

```diff
@@ -298,14 +337,42 @@ void OnGUI ()
 				timeNow = timeNowNew;
 				GetTime();  // Update bet time 
 
+				if (timeRemain > timePrevious) {
+					timePrevious = timeRemain;
+					countPrevious = 5;    // cycles to play the round
+				}
+				timePrevious = timeRemain;
+				countPrevious--;
+				if (countPrevious == 0) {
+					audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/startRound"));
+				}
+
+				if (timeRemain > 15) {
+					alarm10Sec = false;
+					alarm5Sec = false;
+				}
+
+				if ((timeRemain <= 10) && !alarm10Sec) {
+					alarm10Sec = true;
+					audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_10sec"));
+				}
+
+				if ((timeRemain <= 5) && !alarm5Sec) {
+					alarm5Sec = true;
+					audio.PlayOneShot ((AudioClip)Resources.Load ("Audio/alarm_5sec"));
+				}
+					
+
 				// On the multiples of 5 seconds, get the names
```

Comment:
This code is important as it reminds me of the past architecture. I may have to go back to that.